modules/administration/pages/removing-channels.adoc

```diff
@@ -117,25 +110,33 @@ spacewalk-remove-channel -a sles12-sp5-pool-x86_64
 //----
 
 
-==== Remove Packages from the Database via the {webui}
 
-.Procedure: Remove Packages from the Database via the {webui}
-. After removing the channel, you need to delete the packages from the database via the {webui}.
-. In the {productname} {webui} navigate to menu:SUSEManager[Software > Manage > Packages].
+== Remove Packages from the Database and the Filesystem
+
+To finally gain space remove the packages from the database and filesystem.
+
+
+
+=== Remove Packages from the Database Using {webui}
+
+.Procedure: Removing Packages from the Database Using {webui}
+. After removing the channel, you need to delete the packages from the database using the {webui}.
+. In the {productname} {webui}, navigate to menu:SUSEManager[Software > Manage > Packages].
 . Select [literal]``Package in no channel``, and delete the packages.
```

Comment:
spacewalk-remove-channel automatically remove also the packages. This chapter is not needed.
It removes the metadata from DB as well as the files from disk.

This is just a general information in case of other actions remove a package from a channel and do not delete it from disk. But for spacewalk-remove-channel it is not needed.

Reply:
ok, changed accordingly.